### DISCUSSION TITLE
useExternal是否可以在组件卸载时，不移除加载进来的js。

### DISCUSSION THREAD

**Discussion by candy4290** [alibaba/hooks#2163] [Q&A] [answered]
场景：我希望自从第一次加载了该资源后。不再移除该资源，这样在需要再次使用该资源时，也就不会重复加载这个资源。
有什么办法吗？

**Comment by candy4290** [comment]
是否可以提供一个option
参数-说明-类型-默认值
keepWhenUnused-在不持有资源的引用后，保持资源-boolean-默认值false。
如果可以这样做的话，我可以尝试提交一个PR

**Reply by liuyib** [reply, marked as answer]
可以支持，给个 PR 吧，我把这个讨论转成 issue 了：https://github.com/alibaba/hooks/issues/2164
